<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QWidget" name="menu">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QPushButton" name="newGameButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>351</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>New Game</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quitButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>351</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
     <widget class="QLabel" name="background">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>801</width>
        <height>601</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/assets/beach.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>691</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/assets/beachBrawlLogo.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>background</zorder>
     <zorder>newGameButton</zorder>
     <zorder>quitButton</zorder>
     <zorder>label</zorder>
    </widget>
    <widget class="QWidget" name="page">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QLabel" name="background_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>601</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/assets/beach.png);</string>
      </property>
      <property name="lineWidth">
       <number>8</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="FightButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>530</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Fight</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rightArrow">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>440</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>rightArrow</string>
      </property>
     </widget>
     <widget class="QPushButton" name="leftarrow">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>440</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>leftArrow</string>
      </property>
     </widget>
     <widget class="QLabel" name="playerOneSelect">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>420</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="lineWidth">
       <number>8</number>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="characterImg">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>251</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/assets/lifegaurd.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="mainMenuBtn">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>530</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="background_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>601</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/assets/beach.png);</string>
      </property>
      <property name="lineWidth">
       <number>8</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>311</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Chalkboard SE</family>
        <pointsize>64</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(183, 41, 56);</string>
      </property>
      <property name="text">
       <string>WINNER!</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="winnerName">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>280</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Chalkboard SE</family>
        <pointsize>52</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(183, 41, 56);</string>
      </property>
      <property name="text">
       <string>WINNER!</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="newGameBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>390</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>New Game</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitBtn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>390</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
